### PR TITLE
Add devtools hook

### PR DESCRIPTION
**Pull request by TrySound**
I made [devtools](https://github.com/TrySound/derivable-devtools) (docs wip) for derivable but integration requires hook.

## Usage

```js
import 'derivable-devtools'; // should be placed before to initialize hook
import { atom } from 'derivable';
```

screen shot 2017-11-07 at 9 56 51 pm

/cc @andreypopp

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ export var atom = derivable.atom;
 export var atomic = derivable.atomic;
 export var atomically = derivable.atomically;
 export var proxy = derivable.proxy;
-export var derive = derivable.derive;
```
Comment (ds300) [inline src/index.js]: Is this a mistake?

Comment (TrySound) [reply src/index.js]: Nope. It was duplicated. And rollup is failed on it.

**Comment by TrySound** [comment]
The next step is babel plugin with extracted names from the code. And maybe component grouping for react.

File: src/index.js (hunk 1 above)
Comment (TrySound) [reply src/index.js]: Here's duplication
https://github.com/ds300/derivablejs/blob/master/src/index.js#L24-L29

**Pull request merged**